### DISCUSSION TITLE
Documentation Self-hosted  repositories contains invalid value for bumpVersion

### DISCUSSION THREAD

**Discussion by mratwork-dev** [renovatebot/renovate#32006] [Request Help] [answered]
### How are you running Renovate?

None

### Please tell us more about your question or problem

On https://docs.renovatebot.com/self-hosted-configuration/#repositories I read:

Elements in the repositories array can be an object if you wish to define more settings:

```
{
  repositories: [{ repository: 'g/r1', bumpVersion: true }, 'g/r2'];
}
```

In the example the value **true** is used, but that version is not valid according to the documentation of bumpVersion itself.

https://docs.renovatebot.com/configuration-options/#bumpversion

So either this documentation is confusing, or the documentation on bumpVersion is incomplete.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Docs fix: https://github.com/renovatebot/renovate/pull/32008
